src/components/Calculator.js

```diff
@@ -1,47 +1,62 @@
+  }
+
+  handleClick(e) {
+    const value = e.target.innerHTML;
+    const obj = calculate(this.state, value);
+    this.setState(obj);
   }
 
   render() {
+    const { total } = this.state;
     return (
       <div className="container">
         <table className="calculator">
           <tbody className="btns-group">
             <tr className="result-container">
-              <th colSpan={4} className="result">{this.result}</th>
+              <th colSpan={4} className="result">
+                { total || 0 }
+              </th>
             </tr>
             <tr className="btns-row">
-              <td>AC</td>
-              <td>+/-</td>
-              <td>%</td>
-              <td className="orange">÷</td>
+              <td onClick={this.handleClick} aria-hidden="true">AC</td>
+              <td onClick={this.handleClick} aria-hidden="true">+/-</td>
+              <td onClick={this.handleClick} aria-hidden="true">%</td>
+              <td onClick={this.handleClick} className="orange" aria-hidden="true">÷</td>
             </tr>
             <tr className="btns-row">
-              <td>7</td>
-              <td>8</td>
-              <td>9</td>
-              <td className="orange">x</td>
+              <td onClick={this.handleClick} aria-hidden="true">7</td>
+              <td onClick={this.handleClick} aria-hidden="true">8</td>
+              <td onClick={this.handleClick} aria-hidden="true">9</td>
+              <td onClick={this.handleClick} className="orange" aria-hidden="true">x</td>
             </tr>
             <tr className="btns-row">
-              <td>4</td>
-              <td>5</td>
-              <td>6</td>
-              <td className="orange">-</td>
+              <td onClick={this.handleClick} aria-hidden="true">4</td>
+              <td onClick={this.handleClick} aria-hidden="true">5</td>
+              <td onClick={this.handleClick} aria-hidden="true">6</td>
+              <td onClick={this.handleClick} className="orange" aria-hidden="true">-</td>
             </tr>
             <tr className="btns-row">
-              <td>1</td>
-              <td>2</td>
-              <td>3</td>
-              <td className="orange">+</td>
+              <td onClick={this.handleClick} aria-hidden="true">1</td>
+              <td onClick={this.handleClick} aria-hidden="true">2</td>
+              <td onClick={this.handleClick} aria-hidden="true">3</td>
+              <td onClick={this.handleClick} className="orange" aria-hidden="true">+</td>
             </tr>
             <tr className="btns-row">
-              <td colSpan={2}>0</td>
-              <td>.</td>
-              <td className="orange">=</td>
+              <td onClick={this.handleClick} colSpan={2} aria-hidden="true">0</td>
+              <td onClick={this.handleClick} aria-hidden="true">.</td>
+              <td onClick={this.handleClick} className="orange" aria-hidden="true">=</td>
             </tr>
           </tbody>
         </table>
```

Comment:
- [ ] Good job so far :clap:  but I am able to perform math operations with your calculator. Please make sure to implement all calculator operations and fix the click event for all buttons. as required for this project. See more details on the video I recorded for you below :arrow_down: 
https://www.loom.com/share/b233e17c1e2f413b99a0b71895712a0d